holoviews/operation/datashader.py

```diff
@@ -259,6 +277,8 @@ def _process(self, element, key=None):
             categories = array.shape[-1]
             if not self.p.cmap:
                 pass
+            elif isinstance(self.p.cmap, dict):
+                shade_opts['color_key'] = self.p.cmap
             elif isinstance(self.p.cmap, Iterable):
                 shade_opts['color_key'] = [c for i, c in
                                            zip(range(categories), self.p.cmap)]
```

Comment:
We don't have a default ``cmap`` so it just ends up using the appropriate datashader default.